test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -3026,13 +3027,13 @@ public void Mesh2dUndoThenRedoTwoDeleteNodesThroughApi()
                     // Re-do
                     bool redone = false;
 
-                    Assert.AreEqual(0, api.RedoState(ref redone));
+                    Assert.AreEqual(0, api.RedoState(ref redone, ref meshKernelId));
```

Comment:
add an assert that the ref argument has changed meshKernelId from its initial value.